src/scss/atomic/atoms/_preferences-checkbox.scss

```diff
@@ -0,0 +1,91 @@
+/** @format */
+// restored original file _input-checkbox from 2018-05-15 Plas
+// modified it so it only applies to preferences-bar
+@import "~scss/settings/_input.scss";
+@import "~scss/settings/_variables.scss";
+@import "~scss/tools/mixins/_helpers.scss";
+
+$checkboxSize: $buildingUnit * 2;
+$checkmarkSize: $buildingUnit * 1.5;
+
+.preferences-bar [type="checkbox"] + label {
+    display: inline-block;
+    font-size: 14px;
+    line-height: 21px;
```

Comment:
```suggestion
    line-height: 1.5;
```

Reply:
ja, they're all crazy. it's a straight copy paste of plas' original code altered to make it component specific. the PreferencesBar needs refactoring - there are other issues with it and i want to leave those for another day.